lite-variables.tf

```diff
@@ -53,36 +53,29 @@ variable "aks_admin_ip_cidr_map" {
 variable "aks_node_sizes" {
   description = "The VM size for each node of each node pool in the HPCC Systems. Example format is '{ roxie = \"xlarge\", serv = \"2xlarge\", spray = \"xlarge\", thor = \"xlarge\" }'.\nSee https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general for more information."
   type = object({
```

Comment:
Please break this object out into scalar variables (one of the original design choices in this project), with default values for each.  Remember to update the documentation as well.

Reply:
fixed

Reply:
fixed